Library/DiscUtils.Core/Internal/Utilities.cs

```diff
@@ -242,6 +242,26 @@ public static string GetFileFromPath(string path)
             return trimmed.Substring(index + 1);
         }
 
+
+        /// <summary>
+        /// Extracts the extension part of a file from a full path including
+        /// the dot.
+        /// </summary>
+        /// <param name="path">The path to process.</param>
+        /// <returns>The file extension.</returns>
+        public static string GetExtFromPath(string path)
```

Comment:
Out of curiosity, how is this different from `Path.GetExtension`?

Reply:
I think I added this to provide compatibility with .NET standard. I came to the conclusion it was needed because the function above `GetFileFromPath` which does the same as `Path.GetFileName`

Reply:
`GetFileFromPath` was first introduced here: https://github.com/DiscUtils/DiscUtils/blob/521f30a771d1353d8f3291baedee83a5a0da32ad/src/Utilities.cs#L624

Reply:
@qmfrederik ...and since you asked how it it different, here is the Microsoft implementation which isn't particularly beautiful ;) but is probably more robust when given a full path, instead of a filename.

https://github.com/Microsoft/referencesource/blob/master/mscorlib/system/io/path.cs#L300